types/src/transaction/mod.rs

```diff
@@ -1524,6 +1524,15 @@ impl AccountTransactionsWithProof {
     }
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+#[cfg_attr(any(test, feature = "fuzzing"), derive(Arbitrary))]
+pub struct OrderedSignedUserTransaction {
```

Comment:
It is not batched (i.e. this is not a group), but part of a group. I also named it this way in case anyone wants to add other information inside, like position in block, or something?